modules/ca/src/tools/tool_lib.h

```diff
@@ -49,6 +51,7 @@
 
 #define DEFAULT_CA_PRIORITY 0  /* Default CA priority */
 #define DEFAULT_TIMEOUT 1.0     /* Default CA timeout */
+#define DEFAULT_TIMEOUT_ENV "EPICS_CLI_TIMEOUT" /* CA timeout environment var */
```

Comment:
Rename macro to `EPICS_CLI_TIMEOUT`

Reply:
The macro was removed because the code common to the tool apps that referenced that macro was moved to this file. The environment variable now only appears in one place.